saleor/warehouse/management.py

```diff
@@ -198,12 +211,20 @@ def increase_stock(
     function increase `quantity_allocated`. If allocation does not exist function
     create a new allocation for this order line in this stock.
     """
+    plugins_manager = get_plugins_manager()
+    product_variant_stock_trigger = ProductVariantStockWebhookTrigger()
     stock = (
         Stock.objects.select_for_update(of=("self",))
         .filter(warehouse=warehouse, product_variant=order_line.variant)
         .first()
     )
     if stock:
+        product_variant_stock_trigger.append_stock_data(
```

Comment:
`back_in_stock` should be called for both case. If new stock is created for released variant, it means that it is returned to stock.